.github/workflows/wheels.yml

```diff
@@ -77,11 +79,17 @@ jobs:
         # should also be able to do multi-archs on a single entry, e.g.
         # [windows-2019, win*, "AMD64 x86"]. However, those two require a different compiler setup
         # so easier to separate out here.
-        - [ubuntu-22.04, manylinux, x86_64]
-        - [ubuntu-22.04, musllinux, x86_64]
-        - [macos-11, macosx, x86_64]
-        - [macos-14, macosx, arm64]
-        - [windows-2019, win, AMD64]
+        - [ubuntu-22.04, manylinux, x86_64, "", ""]
+        - [ubuntu-22.04, musllinux, x86_64, "", ""]
+        - [macos-13, macosx, x86_64, openblas, "10.9"]
+        # Build x86 Accelerate wheel targeting macOS 13 just for testing
+        - [macos-13, macosx, x86_64, accelerate, "13.0"]
```

Comment:
This entry can be removed if we decide to follow Numpy in not testing the x86 Accelerate wheel.

Reply:
I don't think that this entry should be present. We're only going to use accelerate for macos > 13.0.

Reply:
This was the only way I could think of to test Accelerate wheels built on x86. I agree with you though. The results from this test are mostly meaningless because they do not directly test the wheel we would ship (who knows what changing the deployment target does to the wheel). Furthermore, at the moment, the only failures I observed are known issues (linked in PR description). I'm only worried that new bugs might go uncaught unless we have a proper way to test the x86 Accelerate wheels.

Reply:
I think we should be able to build and test macos_x86_64 on a macos-14 image, even though it uses M1 processors. Let's try that.

Reply:
So remove this line anyway.